platform/stm32f0xx/platform.mk

```diff
@@ -27,10 +27,12 @@ CFLAGS := -Wall -Werror -g -Os -std=c99 -Wno-unused-variable -pedantic \
           -ffunction-sections -fdata-sections -fno-builtin -flto \
           --specs=nosys.specs --specs=nano.specs \
           $(ARCH_CLAGS) $(addprefix -D,$(CDEFINES))
-POST_CFLAGS :=
 
 # Linker flags
 LDFLAGS := $(CLFLAGS) -L$(LDSCRIPT_DIR) -Tstm32f0.ld -fuse-linker-plugin
+# Library functions are ignored unless they are referenced previously
+# so we need to add them at the end, after the gcc command
+LDADD :=
```

Comment:
I'm not sure I understand why we need this LDADD variable. It doesn't work if you put it in LDFLAGS?